ci-operator/step-registry/openshift/e2e/aws/heterogeneous/conf/openshift-e2e-aws-heterogeneous-conf-commands.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# List of exclude tests from conformance/serial suite
+if [ "${HETEROGENEOUS:-}" = 'true' ]; then
+    cat > "${SHARED_DIR}/excluded_tests" << EOF
+"[sig-apps][Feature:DeploymentConfig] deploymentconfigs  should adhere to Three Laws of Controllers [Skipped:Disconnected] [Suite:openshift/conformance/parallel]"
+"[sig-apps][Feature:DeploymentConfig] deploymentconfigs adoption will orphan all RCs and adopt them back when recreated [Skipped:Disconnected] [Suite:openshift/conformance/parallel]"
```

Comment:
This feels like a brittle approach.  Is there something tracking teaching the origin suite to handle this, either by making the test-cases robust enough to work on heterogeneous clusters, or by teaching them to skip automatically on heterogeneous clusters?

Reply:
Ah, your later "This is temporary until we support manifest list for internal registry." suggests that will make the cluster robust enough to run these tests, and my ask is just extending that `documentation` string to include a tracker ref for that effort.